pkg/transformer/utils.go

```diff
@@ -112,6 +120,77 @@ func ParseVolume(volume string) (name, host, container, mode string, err error)
 	return
 }
 
+func parseWindowsVolume(volume string) (name, host, container, mode string, err error) {
```

Comment:
add example volume string in the comment so others can easily understand this function

Reply:
done